devito/ops/operator.py

```diff
@@ -24,6 +25,15 @@ def _specialize_iet(self, iet, **kwargs):
         ops_timing = Call(namespace['ops_timing_output'], [FunctionPointer("stdout")])
         ops_exit = Call(namespace['ops_exit'])
 
+        dims = []
+        for n, (section, trees) in enumerate(find_affine_trees(iet).items()):
+            dims.append(len(trees[n].dimensions))
+
+        assert (d == dims[0] for d in dims), "The OPS backend currently assumes that all kernels have the same number of dimension"
+        
+        self._includes.append('stdio.h')
```

Comment:
Replace "_includes" to "_default_includes" and move to __init__

Reply:
tried that, but didn't work